### PR TITLE
ACL update/revision Activities/small issue fix in company review

### PR DESCRIPTION
**Pull request by choutha**
added version of Activity --> not that important to rest of project. This is a simple version with key functionallity

changed isAllowed function in AbstractAclService --> important since it is about general authorization. Changed the method such that it checks the actual roles instead of the user object against several role strings

changed the approver variable in CompanyReview --> this was giving errors when doctrine was creating the database. Now changed the relation to a OneToMany relation as it should be (discussed with Geert)

### CODE REVIEW

File: module/Activity/src/Activity/Model/ActivitySignup.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    /**
+     * Who is subscribed
+     *
+     * @ORM\Column(nullable=false)
+     * @ORM\ManyToOne(targetEntity="Activity\Model\Activity", inversedBy="roles")
+     * @ORM\JoinColumn(referencedColumnName="id")
+     */
+    protected $activity_id;
+
```
Comment (jszanto) [inline module/Activity/src/Activity/Model/ActivitySignup.php]: De casing hier zou volgens mij camelCase moeten zijn om consistent te blijven met de rest.

Comment (choutha) [inline module/Activity/src/Activity/Model/ActivitySignup.php]: You mean it should be $activityId instead of $activity_id?

**Comment by choutha** [comment]
Already other pull request ready with a full implementation of organ ACL working. Not yet uploaded since this one first needs to be accepted

File: module/Activity/src/Activity/Model/ActivitySignup.php (hunk 1 above)
Comment (jszanto) [inline module/Activity/src/Activity/Model/ActivitySignup.php]: yes I think so.

File: module/Application/src/Application/Service/AbstractAclService.php [hunk 2]
```diff
@@ -34,27 +34,69 @@ public function getRole()
...
+		
+		var_dump($roles);
+		
+		foreach($roles as $role){
+			if($this->getAcl()->isAllowed($role, $resource, $operation)){
+				return true;
+			}
+		}
+		return false;
+    }
```
Comment (kokx) [inline module/Application/src/Application/Service/AbstractAclService.php]: Dit is dus juist fout. Ten eerste staat er nog een `var_dump` in. Maar vooral, de roles van een user zijn al goed gezet in de ACL, waardoor dit helemaal onnodig is. Zie https://github.com/GEWIS/gewisweb/blob/master/module/User/Module.php#L154

**Comment by jszanto** [comment]
Vergeet niet om alle tekst in de views door translate() heen te halen.

**Pull request closed without merging**